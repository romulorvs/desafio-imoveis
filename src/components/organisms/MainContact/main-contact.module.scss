/**
 * IMPORTS
 */
 @import '../../../styles/mixins.scss';

/**
 * CONTENT
 */

.main {
  @include container(700px);
  flex-grow: 1;
  display: flex;
  justify-content: center;
  position: relative;
  flex-direction: column;

  section {
    display: flex;
    flex-direction: column;
    padding: 16px;
    border-radius: 8px;
    background-color: white;
    box-shadow: 0px 2px 1px rgba(0, 0, 0, 0.25), 0px 4px 4px rgba(0, 0, 0, 0.15);

    h1 {
      font-size: 2.8rem;
      border-bottom: 1px solid #e4e4e4;
      padding-bottom: 12px;
      margin-bottom: 16px;
      font-weight: 500;
    }

    h2 {
      margin-bottom: 16px;
      font-size: 1.6rem;
      color: #676767;
      font-weight: 300;
      border-bottom: 1px solid #e4e4e4;
      padding-bottom: 12px;
      margin-bottom: 16px;
    }

    li {
      margin-left: 16px;
      font-size: 1.4rem;
      color: #9c9c9c;
    }

    li + li {
      margin-top: 8px;
    }

    a {
      color: #1ea0ff;
      font-weight: bold;
    }
  }
}