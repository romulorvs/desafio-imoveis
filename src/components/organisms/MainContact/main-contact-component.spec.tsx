import { render } from '@testing-library/react'

import MainContact from './main-contact-component'

describe('MainContact Component', () => {
  test('Should display the component correctly.', () => {
    const { container } = render(<MainContact />)

    expect(container.innerHTML).toMatchSnapshot()
  })
})
