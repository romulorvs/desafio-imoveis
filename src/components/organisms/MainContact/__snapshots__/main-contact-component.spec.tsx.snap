// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContact Component Should display the component correctly. 1`] = `"<main class=\\"main\\"><section><h1>Contato</h1><h2>Esta aplicação foi criada como um desafio do processo seletivo da OLX. Você pode encontrar o repositório e o meu contato no LinkedIn abaixo.</h2><ul><li>Repositório: <a href=\\"https://github.com/romulorvs/eng-zap-challenge-javascript\\" target=\\"_blank\\" rel=\\"noreferrer\\">github.com/romulorvs/eng-zap-challenge-javascript</a></li><li>Linkedin: <a href=\\"https://www.linkedin.com/in/dev-romulo-vieira/\\" target=\\"_blank\\" rel=\\"noreferrer\\">linkedin.com/in/dev-romulo-vieira/</a></li></ul></section></main>"`;
